streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -271,11 +274,16 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
   }
 
   /** Check if a sequence of numbers is in increasing order */
-  def isInIncreasingOrder(seq: Seq[Long]): Boolean = {
-    for (i <- 1 until seq.size) {
-      if (seq(i - 1) > seq(i)) {
+  def isInIncreasingOrder(data: Iterable[Long]): Boolean = {
```

Comment:
I feel like this can be a Scala one liner by mapping to pairs of successive elements and calling exists or something to detect a misordering. Minor, not sure as I'm not at a keyboard to check it